includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +571,47 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if (
+			self::is_stories_experience_enabled() &&
+			(
+				'edit-amp_story' === get_current_screen()->id ||
+				'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id
+			)
+		) {
+			printf(
+				'<div class="notice notice-warning"><p>%s</p></div>',
+				esc_html__( 'The Stories experience in the AMP plugin has been deprecated and will no longer be supported. A separate plugin for Stories be available soon for testing. Please backup or export your existing Stories as they will not be available in the next version of the AMP plugin.', 'amp' )
+			);
+		} elseif ( ! self::is_stories_experience_enabled() && 'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id ) {
+			printf(
+				'<div class="notice notice-info"><p>%s</p></div>',
+				esc_html__( 'The Stories experience has been removed from the AMP plugin. This beta feature is being split into a separate standalone plugin which will be available for installation soon.', 'amp' )
+			);
+		}
+	}
+
+	/**
+	 * Render the Stories deprecation notice in the Story editor.
+	 */
+	public static function render_stories_deprecation_editor_notice() {
+		$script = sprintf(
+			"( function( wp ) {
+						wp.data.dispatch( 'core/notices' ).createNotice(
+							'warning',
+							'%s',
+							{ isDismissible: false }
+						);
+					} )( window.wp );",
+			esc_html__( 'The Stories experience in the AMP plugin has been deprecated and will no longer be supported. A separate plugin for Stories be available soon for testing. Please backup or export your existing Stories as they will not be available in the next version of the AMP plugin.', 'amp' )
```

Comment:
Also, instead of `'%s'` this should be just `%s` and instead of `esc_html__()` we need to use `wp_json_encode( __( ... ) )`

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4219/commits/3e55b5ed4d6b2d8e42e0072636de0b717fe8c9ca.